js/mexc3.js

```diff
@@ -1623,18 +1623,20 @@ module.exports = class mexc3 extends Exchange {
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} params extra parameters specific to the mexc3 api endpoint
+         * @param {string|undefined} params.marginMode only 'isolated' is supported for spot-margin trading
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
+        const [ marginMode, query ] = this.handleMarginModeAndParams ('createOrder', params);
```

Comment:
@Dan-krm currently we're not handling `margin` in a unified way for swap orders, can you add it? (for the sake of consistency)

We can make use of the default `handleMarginMode` method inside `createOrder` (to avoid repeating this logic twice), but bear in mind that the isolated only restriction does not apply to derivatives

Reply:
Yeah, I'll add handling margin in a unified way to createSwapOrder as well!